backend/utils.py

```diff
@@ -10,6 +12,44 @@
 
 JSON_TYPE = Union[Dict, List]
 
+def commify(n):
+    """Those dirty commies
+    ░░░░░░░░░░▀▀▀██████▄▄▄░░░░░░░░░░
+    ░░░░░░░░░░░░░░░░░▀▀▀████▄░░░░░░░
+    ░░░░░░░░░░▄███████▀░░░▀███▄░░░░░
+    ░░░░░░░░▄███████▀░░░░░░░▀███▄░░░
+    ░░░░░░▄████████░░░░░░░░░░░███▄░░
+    ░░░░░██████████▄░░░░░░░░░░░███▌░
+    ░░░░░▀█████▀░▀███▄░░░░░░░░░▐███░
+    ░░░░░░░▀█▀░░░░░▀███▄░░░░░░░▐███░
+    ░░░░░░░░░░░░░░░░░▀███▄░░░░░███▌░
+    ░░░░▄██▄░░░░░░░░░░░▀███▄░░▐███░░
+    ░░▄██████▄░░░░░░░░░░░▀███▄███░░░
+    ░█████▀▀████▄▄░░░░░░░░▄█████░░░░
+    ░████▀░░░▀▀█████▄▄▄▄█████████▄░░
+    ░░▀▀░░░░░░░░░▀▀██████▀▀░░░▀▀██░░
+    """
+    return f'{n:,}'
+
+
+def send_email(subject: str, body: str, to=['sigfried@sigfried.org', 'jflack@jhu.edu']):
```

Comment:
@Sigfried I gave `send_email` a shot, but Gmail doesn't work as of 2022/05: https://support.google.com/accounts/answer/6010255
> To help keep your account secure, from May 30, 2022,Google no longer supports the use of third-party apps or devices which ask you to sign in to your Google Account using only your username and password